lib/Parse/ParseExpr.cpp

```diff
@@ -3557,6 +3559,57 @@ ExprResult Parser::ParseReturnValueExpression() {
   return Actions.ActOnReturnValueExpr(Loc);
 }
 
+/// [Checked C] pack-expression:
+///   _Pack '(' expression ',' type-name ',' type-name ')'
+ExprResult Parser::ParsePackExpression() {
+  assert(Tok.is(tok::kw__Pack) && "Not a pack expression");
+  SourceLocation StartLoc = ConsumeToken();
+
+  if (Tok.isNot(tok::l_paren)) return ExprError();
+  ConsumeParen();
+
+  auto PackedExpr = ParseExpression();
+  if (PackedExpr.isInvalid()) {
+    SkipUntil(tok::r_paren, StopAtSemi);
+    return ExprError();
+  }
+
+  if (ExpectAndConsume(tok::comma)) {
+    SkipUntil(tok::r_paren, StopAtSemi);
+    return ExprError();
+  }
+
+  auto ExistTpe = ParseTypeName();
```

Comment:
Why use the variable name ExistTpe instead of ExistType?

Reply:
Fixed